vaadin-progress-bar.html

```diff
@@ -23,40 +24,28 @@
         position: relative;
       }
 
-      #background {
-        background: var(--vaadin-progress-background-color);
+      #bar {
+        background: var(--vaadin-progress-bar-color);
         height: 100%;
         position: absolute;
         width: 100%;
       }
 
-      #foreground {
-        background: var(--vaadin-progress-foreground-color);
+      #value {
+        background: var(--vaadin-progress-value-color);
         height: 100%;
         position: absolute;
         transform-origin: 0 50%;
         transition: transform 150ms;
         width: 100%;
-      }
-
-      :host(:not([value])) #foreground {
-        animation: indeterminate-progress-bar 1s infinite;
-      }
-
-      @keyframes indeterminate-progress-bar {
-        0% {
-          transform: translateX(-100%);
-        }
-
-        100% {
-          transform: translateX(100%);
-        }
+        transform: scaleX(var(--vaadin-progress-value));
       }
 
     </style>
 
-    <div id="background"></div>
-    <div id="foreground" style$="transform: scaleX([[_normalizeValue(value, min, max)]])"></div>
+    <div id="bar">
+      <div id="value"></div>
+    </div>
```

Comment:
Should use `part` instead of `id`, right? Or is that coming in another PR? In any case, remember to document the stylable parts.

Reply:
That will be done in a separate PR, right?